README.md

```diff
@@ -189,3 +195,36 @@ Note that the double backslash above are just used to escape `\` in the TOML for
 `$\backslash$` does not work, but you can use `$\setminus$` instead.
 
 Only the x86_64 Linux, Windows GNU, and macOS builds have full functionality (matrix, ...) , all other builds have compromised capabilities. See [#39](https://github.com/lzanini/mdbook-katex/issues/39) for the reasons.
+
+## Escaping mode
+
+"Escaping mode" is a beta feature that escapes the string needed for a formula in advance so that it remains the original formula after the md processor. This mode requires client-side js. May be useful for those who have problems with quickjs.
+
+Disable pre-render to use "Escaping mode". Don't forget to include `katex.js` (you can include it in [index.hbs](https://rust-lang.github.io/mdBook/format/theme/index-hbs.html)).
```

Comment:
Suggestion: put some of these in a `<details>` so the README remains somewhat short.

Reply:
There are some parsing issues with the code blocks in the details block, it can't be render.